src/router.js

```diff
@@ -14,7 +14,7 @@ router.post('/register', (req, res, next) => {
 
 // login
 
-router.get('/login', (req, res, next) => {
+router.post('/login', (req, res, next) => {
```

Comment:
It is good to see you changed your mind and used the correct HTTP verb for the /login  router ie. POST rather than GET